app/jobs/scheduled/resend_validation_emails.rb

```diff
@@ -0,0 +1,22 @@
+module Jobs
+  class ResendValidationEmails < Jobs::Scheduled
+    every 2.days
+
+    def execute(args)
+      User.joins("LEFT JOIN user_custom_fields ON users.id = user_id AND user_custom_fields.name = 'activation_reminder'")
+        .where(active: false, user_custom_fields: { value: nil })
+        .where('users.created_at < ?', 2.days.ago)
+        .each do |user|
```

Comment:
We should not be loading too much users with these conditions but it's better to always use `find_each` when a specific order is not required.

```suggestion
        .find_each do |user|
```